cmd/cogs/main.go

```diff
@@ -2,8 +2,29 @@ package main
 
 import (
 	"fmt"
+	"os"
+
+	"github.com/bestowinc/cogs"
```

Comment:
not that it matters, but just curious here. I noticed the actual URL for our github is capitalized on Bestowinc, do we favor one or the other or just  🤷‍♂️

Reply:
what's your preference? I'll go with that.

Reply:
The actual address to our repo lol [https://github.com/Bestowinc/cogs/pull/1](https://github.com/Bestowinc/cogs) personally I prefer how you have it here, but as long as we consistent Idc really